src/api/app/models/event_find_subscriptions.rb

```diff
@@ -8,18 +8,18 @@ def subscriptions
     @payload = @event.payload
     @subscriptions = EventSubscription.where(eventtype: @event.class.classnames)
 
-    # 1. generic defaults
+    # Get the defaults
+    # FIXME: These defaults can only be created by manually adding EventSubscription
+    #        to the database. We don't do that anywhere in the code base. Either
+    #        we'll do an interface for it or set the default somewhere sane...
     @toconsider = @subscriptions.where('user_id is null AND group_id is null').to_a
```

Comment:
We have a defaults scope now on EventSubscription which you would use here like:
```ruby
@toconsider = @subscriptions.defaults
```

Reply:
Same here, I really don't want to change this code. I hope I can 💀 it in the next weeks...